cns/wireserver/net.go

```diff
@@ -79,3 +124,9 @@ func calculateGatewayIP(cidr string) (net.IP, error) {
 	}
 	return gw, nil
 }
+
+func macAddressesEqual(macAddress1, macAddress2 string) bool {
+	macAddress1 = strings.ToLower(strings.ReplaceAll(macAddress1, ":", ""))
+	macAddress2 = strings.ToLower(strings.ReplaceAll(macAddress2, ":", ""))
+	return macAddress1 == macAddress2
```

Comment:
compare doesn't work without replacing ':'?

Reply:
the formats of both MACs in caller func - L68, differ in one being plain & one having ":" in the mac address string